src/components/WebexMeetingGuestAuthentication/__snapshots__/WebexMeetingGuestAuthentication.stories.storyshot

```diff
@@ -36,27 +36,36 @@ Array [
         >
           Your name
```

Comment:
please modify GuestAuth and HostAuth to have the labels sent to InputField instead of putting them separately.